cmd/azure.go

```diff
@@ -29,6 +29,7 @@ import (
 	"net/url"
 	"strconv"
```

Comment:
One more thing to notice here is we are not persisting metadata and can get lost upon gateway restart, so i am not sure why we even need to attempt saving metadata in memory.. ? because if the idea is to address the situation of preserving content-type we should indeed behave like how an S3 would behave.. i.e even upon a future completion of the upload it should save the content-type. 

Otherwise this change makes it inconsistent in nature where users need to be sure that it won't save content-type if server just happens to have been restarted..